apps/src/aichat/views/tabs/TabPanel.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+interface TabPanelProps {
+  content: React.ReactNode;
+  isActive: boolean;
+  id: string;
+  labelledBy: string;
```

Comment:
nit: For design system version of Tabs I'll be writing comments describing every prop, but for now might be good to add a comment describing at least this one maybe?

Reply:
Done!